### DISCUSSION TITLE
SVG to PNG

### DISCUSSION THREAD

**Discussion by 010sfs** [brend8c/FFMPEG_BAT#7] [Q&A] [answered]
I realized that you can not convert raster graphics to vector. Is it possible to do the opposite?
Convert SVG to PNG?

**Comment by brend8c** [comment, marked as answer]
@010sfs, SVG files are vector graphics, you have to rasterize them first. 
FFmpeg doesn't support that unless you compiled FFmpeg with librsvg.
You need to compile FFmpeg from source code with `librsvg` or find a prebuilt one that compiled with `librsvg`.

[Helper script ](https://github.com/rdp/ffmpeg-windows-build-helpers)for cross compiling some media tools for windows, like customizable ffmpeg.exe (with or without non-free components, etc), and some other bonuses like mplayer, mp4box, mxf, etc.

**Comment by 010sfs** [comment]
Oh thank! )
